conans/cli/cli.py

```diff
@@ -24,12 +24,12 @@ class Cli(object):
     help of the tool.
     """
 
-    def __init__(self, conan_api, cli_out=None):
+    def __init__(self, conan_api):
         assert isinstance(conan_api, Conan), "Expected 'Conan' type, got '{}'".format(
             type(conan_api))
         self._conan_api = conan_api
         self._out = conan_api.out
-        self._cli_out = cli_out or CliOutput(color=self._out.color)
+        self._cli_out = CliOutput()
```

Comment:
Can't we simplify this? If this has no state, no definition, no injection, it can be just local?

Reply:
Do you mean to remove the CliOutput class? 
It's just a wrapper to handle colors a bit simpler in the formatters, should I remove that and pass just the stream to the formatters instead of cli_out? (https://github.com/conan-io/conan/blob/2f5978359ba358f3c00af7ebd945dd3fa9d2b898/conans/cli/cli.py#L139-L140) Then the user that writes the formatter handles the color inside of it?